app/controllers/feedbacks_controller.rb

```diff
@@ -0,0 +1,31 @@
+class FeedbacksController < ApplicationController
+  before_action -> { authorize! Feedback }
+
+  def new
+    @feedback = Feedback.new
+  end
+
+  def create
+    result = action
+
+    @feedback = result.either(->(_feedback) { result.value! }, ->(_feedback) { result.failure })
```

Comment:
1. `result.value!` isn't feedback here
2. Why you need `@feedback` in green scenario?

Reply:
In fact I was looking for methods and therefore didn't pay attention to the fact that result.value! returns a Unit Object. Yes, of course, @feedback isn't needed for success.